net/xorp/Makefile.trueos

```diff
@@ -1 +0,0 @@
-USE_OPENRC_SUBR=	openrc-xorp
```

Comment:
Removing that file would broke openrc for this packages

Reply:
There is a lot Makefile.trueos whit USE_OPENRC_SUBR and we need those for openrc.